src/show.jl

```diff
@@ -70,6 +70,11 @@ function print_header(io, Δt, Δb, ∑t, ∑b, name_length, header, allocations
     if header
         time_alloc_pading = " "^(strwidth(sec_ncalls))
 
+        if length(title) > strwidth(sec_ncalls)
+            title = string(title[1:strwidth(sec_ncalls)-3], "...")
```

Comment:
I copied from https://github.com/KristofferC/TimerOutputs.jl/pull/20/files#diff-f296e38172e8384204314bcecde8499aR131 but what would be a better way?